homeassistant/components/fastdotcom/services.py

```diff
@@ -0,0 +1,49 @@
+"""Service for Fast.com integration."""
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.core import HomeAssistant, ServiceCall
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers import issue_registry as ir
+
+from .const import DOMAIN, SERVICE_NAME
+from .coordinator import FastdotcomDataUpdateCoordindator
+
+
+async def async_setup_services(hass: HomeAssistant) -> None:
```

Comment:
This doesn't need to be a coroutine function as there's no await inside. Make it an async callback and decorate it with `@callback` imported from core.py.